test/integration/get_url_path.dbtspec

```diff
@@ -47,7 +47,7 @@ projects:
         }}
         SELECT {{ dbt_utils.get_url_path(field='url') }} AS url_path
         FROM {{ ref('urls') }}
-        WHERE url_path = 'path1/path2/path3#pivot-source'
+        WHERE url_path = '/path1/path2/path3'
```

Comment:
Is this any dummy path?

Reply:
I don't quite understand the question.
In url should look like this
                            .-:80-----.                      
>>-http://--+-host name--+--+---------+--/--path component------><
            '-IP address-'  '-:--port-'                      

>--+-----------------+-----------------------------------------><
   '-?--query string-'   

"A URL can be followed by a fragment identifier. The separator used between the URL and the fragment identifier is the # character. A fragment identifier is used to point a web browser to a reference or function in the item that it has just retrieved. For example, if the URL identifies an HTML page, a fragment identifier can be used to indicate a subsection within the page, using the ID of the subsection" [Ref](https://www.ibm.com/docs/en/cics-ts/5.1?topic=concepts-components-url)

This means the string starting from \# is not part of the path.
Please see below how python urllib parses the same strings
$ python
Python 3.11.6 (main, Oct  3 2023, 02:51:45) [Clang 14.0.3 (clang-1403.0.22.14.1)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> from urllib import parse
>>> parse.urlparse("https://docs.getdbt.com/path1/path2/path3#pivot-source?utm_source=123")                                                                                            ParseResult(scheme='https', netloc='docs.getdbt.com', path='/path1/path2/path3', params='', query='', fragment='pivot-source?utm_source=123')
>>> parse.urlparse("https://google.com/path1/path2/path3#pivot-source?resource_id=https://quickstarts.teradata.com/&utm_source=123&utm_medium=gamma&utm_campaign=wnc_10030322&utm_content=msg_100058679&hl=en")
ParseResult(scheme='https', netloc='google.com', path='/path1/path2/path3', params='', query='', fragment='pivot-source?resource_id=https://quickstarts.teradata.com/&utm_source=123&utm_medium=gamma&utm_campaign=wnc_10030322&utm_content=msg_100058679&hl=en')
>>>